libs/net/dal/Services/ReportService.cs

```diff
@@ -595,23 +599,24 @@ public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = nul
                 var defaultIndex = searchUnpublished ? _elasticOptions.UnpublishedIndex : _elasticOptions.PublishedIndex;
 
                 var content = await _elasticClient.SearchAsync<API.Areas.Services.Models.Content.ContentModel>(defaultIndex, query);
+                var contentHits = content.Hits.Hits.ToArray();
```

Comment:
I'm not certain why this was resulting in failures, but this change resolve the 'Remove Duplicate Content' bug.